code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -1779,3 +1779,25 @@
 	mutated = TRUE
 
 	return ..()
+
+/datum/reagent/consumable/ethanol/voxwine
+	name = "Vox Wine"
+	id = "voxwine"
+	description = "An interesting drink some may find rather offensive."
+	color = "#cdd37b" // rgb: 205, 211, 123
+	alcohol_perc = 0.2
+	drink_icon = "voxwine"
+	drink_name = "Vox Wine"
+	drink_desc = "An interesting drink some may find rather offensive."
+	taste_description = "victory"
+
+/datum/reagent/consumable/ethanol/planet_cracker
+	name = "Planet Cracker"
+	id = "planet_cracker"
+	description = "This jubilant drink celebrates humanity's triumph over the alien menace. May be offensive to non-human crewmembers."
+	color = "#fdb7ff" // rgb: 253, 183, 255
+	alcohol_perc = 0.2
```

Comment:
```suggestion
	alcohol_perc = 0.4
```
Same as with the vox wine, this drink's components are _much_ stronger than the drink itself